packages/lib/src/toolbar/controls/ToggleGroup.tsx

```diff
@@ -66,14 +66,15 @@ function Btn(props: BtnProps) {
 interface Props extends ToggleGroupProps {
   ariaLabel?: string;
   children: ReactElement<BtnProps>[];
+  id?: string;
 }
 
 function ToggleGroup(props: Props) {
-  const { role, ariaLabel, value, disabled, onChange, children } = props;
+  const { role, ariaLabel, value, disabled, onChange, children, id } = props;
 
   return (
     <ToggleGroupContext.Provider value={{ role, value, disabled, onChange }}>
-      <div className={styles.group} role={role} aria-label={ariaLabel}>
+      <div className={styles.group} role={role} aria-label={ariaLabel} id={id}>
```

Comment:
Small change to allow an `id` prop to be used for `htmlFor` on the `label` of the `ToggleGroup`. This follows the pattern of the cell width control.